Uplift/Views/LoadingScreenHomeView.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  LoadingScreenHomeView.swift
+//  Alamofire
+//
+//  Created by Cameron Hamidi on 2/7/20.
+//
+
+import UIKit
+import SnapKit
+
+class LoadingScreenHomeView: UIView {
+
+    let navBarGreyedOut = UIView()
+    let tabBarGreyedOut = UIView()
+    
+    
+
+}
```

Comment:
I feel like we should just put these UI files in the next PR when you actually put stuff inside